src/Mantasflowers.WebApi/Controllers/UserController.cs

```diff
@@ -0,0 +1,226 @@
+﻿using FirebaseAdmin.Auth;
+using Mantasflowers.Persistence.Authentication;
+using Mantasflowers.WebApi.Extensions;
+using Mantasflowers.WebApi.Responses;
+using Mantasflowers.WebApi.Setup.Authentication;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Serilog;
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Security.Claims;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Mantasflowers.WebApi.Controllers
+{
+    [ApiController]
+    [Route("/firebase")]
```

Comment:
I would just name this "/authentication". The user of the API doesn't care what's the actual auth provider.

Reply:
Endpoints defined in /firebase route are not only about authentication. As was mentioned in the comment, can also, for example, create a user - basic CRUD operations.